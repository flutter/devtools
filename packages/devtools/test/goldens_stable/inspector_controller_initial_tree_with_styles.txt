▼[R][root]
  ▼[M]MyApp
    ▼[M]MaterialApp
      ▼[S]Scaffold
      ├───▼[C]Center
      │     [/icons/inspector/textArea.png]Text
      └─▼[A]AppBar
          [/icons/inspector/textArea.png]Text
