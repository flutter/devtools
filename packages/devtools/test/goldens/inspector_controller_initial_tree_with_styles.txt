▼[R]<grayed> </grayed>root <grayed>]</grayed>
  ▼[M]<grayed> </grayed>MyApp
    ▼[M]<grayed> </grayed>MaterialApp
      ▼[S]<grayed> </grayed>Scaffold
      ├───▼[C]<grayed> </grayed>Center
      │     ▼[/icons/inspector/textArea.png]<grayed> </grayed>Text
      └─▼[A]<grayed> </grayed>AppBar
          ▼[/icons/inspector/textArea.png]<grayed> </grayed>Text
